reference/release-notes/release-notes.md

```diff
@@ -10,31 +10,29 @@ The following dates for upcoming releases are not final and could vary depending
 
 | Tentative Release date | Planned release version |
 |----|----|
-| August 14 2024 | Anbox Cloud 1.23.0 |
 | September 18 2024 | Anbox Cloud 1.23.1 |
 | October 16 2024 | Anbox Cloud 1.23.2 |
+| November 14 2024 | Anbox Cloud 1.24.0 |
 
 ## Recent releases
 
 | Release date   |  Release notes  |
 |----|----|
-| July 18 2024 | [Anbox Cloud 1.22.2](https://discourse.ubuntu.com/t/46495) |
+| August 14 2024 | [Anbox Cloud 1.23.0](1.23.0.md) |
+| July 18 2024 | [Anbox Cloud 1.22.2](1.22.2.md) |
 | June 13 2024 | [Anbox Cloud 1.22.1](1.22.1.md) |
 | May 15 2024 | [Anbox Cloud 1.22.0](1.22.0.md) |
 
-### What's new in 1.22.x?
+### What's new in 1.23.x?
 
 Along with bug fixes and general improvements, Anbox Cloud 1.22.x includes:
```

Comment:
Shouldn't this be:
```suggestion
Along with bug fixes and general improvements, Anbox Cloud 1.23.x includes:
```
There might be other 1.22.x leftovers, so maybe a search across files could help to spot them all. :)

Reply:
Good catch, thanks!